### DISCUSSION TITLE
How to use a specific version of OpenCover when multiple are installed.

### DISCUSSION THREAD

**Discussion by table-apple** [OpenCover/opencover#1046] [Q&A]
In my one of the .cake script (1), I have specified the version of OpenCover to be installed using the tool directive. 
#tool "nuget:?package=OpenCover&version=4.7.1204"
This installs the specified version of OpenCover in the tools section.

Another .cake script (2), is installing the latest version of OpenCover in the tools section.
#tool "nuget:?package=OpenCover"

I want to use OpenCover in the .cake script (1) (as below). How do I specify which version to use? By default, its using the OpenCover version installed by the command - #tool "nuget:?package=OpenCover"

Executing -
 OpenCover(
      // Body
);

Please look at the tools directory, for getting an idea on the directory structure.

![image](https://user-images.githubusercontent.com/22478482/124490805-2ba37200-ddd0-11eb-9907-0c307aecbd6f.png)

**Comment by sawilde** [comment]
i don't know much about the cake script but I suspect you would need to override its default location somehow
